src/main/java/org/apache/commons/lang3/StringUtils.java

```diff
@@ -221,6 +226,27 @@ public static String abbreviate(final String str, final int maxWidth) {
         return abbreviate(str, "...", 0, maxWidth);
     }
 
+    /**
+     * <p>Removes any number contained in a String using Regex.</p>
+     *
+     * <pre>
+     * StringUtils.removeAllNumbers(null)        = ""
+     * StringUtils.removeAllNumbers("")          = ""
+     * StringUtils.removeAllNumbers(" ")         = ""
+     * StringUtils.removeAllNumbers("123abc")    = "abc"
+     * StringUtils.removeAllNumbers("123a b c")  = "a b c"
+     * StringUtils.removeAllNumbers("abc123")    = "abc"
+     * StringUtils.removeAllNumbers("123abc123") = "abc"
+     * StringUtils.removeAllNumbers("a1b2c3")    = "abc"
+     * </pre>
+     * 
+     * @param str String to check, may be null
+     * @return The String without the numbers
+     */
+    public static String removeAllNumbers(String str) {
+      return !isBlank(str) ? str.replaceAll(REGEX_FIND_NUMBERS, EMPTY) : EMPTY;
```

Comment:
1. if isBlank(str) should not return EMPTY, but return str itself.
2. maybe using a hand-write loop will be faster than using regex. please do a benchmark.

Reply:
Hello XenoAmess!

My English is unfortunately still intermediate, did you suggest that I do this validation manually?

Was it something like that?

```
public static String removeAllNumbers(String str) {
        String result = "";
        if(!isEmpty(str)) {
          char[] array = str.toCharArray();
          for(int i = 0 ; i < array.length; i++) {
              if(!Character.isDigit(array[i])) {
                result += array[i];
              }
          }
          return result;
        }
        return str;
}
```

Reply:
XenoAmess I am creating a new commit with this change. I also corrected the checkstyle. Could you review again?

Reply:
@muriloalvesdev Hi.
I re-reviewed this pr and have no more suggestions about the implementation.
Please open a JIRA ticket for this pr, and add the string of that ticket in front of the pr's name 
(example: https://github.com/apache/commons-lang/pull/524)

Next step is waiting for some people who have merge access(means, Committer) to review this pr, and see if we should merge this.